image/manifest.go

```diff
@@ -90,10 +88,11 @@ func manifestInstanceFromBlob(src types.ImageSource, manblob []byte, mt string)
 		return manifestSchema2FromManifest(src, manblob)
 	case manifest.DockerV2ListMediaType:
 		return manifestSchema2FromManifestList(src, manblob)
-	case "":
-		return nil, errors.New("could not guess manifest media type")
 	default:
-		return nil, fmt.Errorf("unsupported manifest media type %s", mt)
+		// if it's not a recognized manifest media type we'll try the last time
+		// to deserialize using v2s1 as per https://github.com/docker/distribution/blob/master/manifests.go#L108
+		// and https://github.com/docker/distribution/blob/master/manifest/schema1/manifest.go#L50
+		return manifestSchema1FromManifest(manblob)
```

Comment:
… still, behaving the same as docker/distribution is valuable in either case.

It would be nice to preserve the Crane comment, perhaps here, replacing the delete comment in the other file (just noting that Crane-maintained repos can report various bogus MIME types and link to that bz, perhaps).